distribution/packages/src/rpm/wazuh-indexer.rpm.spec

```diff
@@ -264,7 +264,9 @@ exit 0
 %attr(750, %{name}, %{name}) %{product_dir}/performance-analyzer-rca/bin/*
 
 %changelog
-* Fri Jul 12 2024 support <info@wazuh.com> - 4.9.0
+* Fri Aug 15 2024 support <info@wazuh.com> - 4.9.1
+- More info: https://documentation.wazuh.com/current/release-notes/release-4-9-1.html
+* Fri Aug 15 2024 support <info@wazuh.com> - 4.9.0
```

Comment:
Friendly reminder: the day should be aligned with the date. Aug 15 is Thu. This may cause RPM package error